.goreleaser.yml

```diff
@@ -15,6 +15,29 @@ archives:
       - completions/*
       - manpages/*
 
+aurs:
+  - homepage:  https://www.updatecli.io
+    description: Updatecli is a declarative dependency management command line tool.
+    maintainers:
+      - 'Olivier Vernin <olivier at vernin dot me>'
```

Comment:
You can also add me here :smile: : `Jonathan Neidel <aur at jneidel dot com>`

Reply:
Awesome, I was looking for that email \o/

Reply:
```suggestion
      - 'Olivier Vernin <olivier at vernin dot me>'
      - 'Jonathan Neidel <aur at jneidel dot com>'
```